Apps/Sandcastle/gallery/3D Tiles Point Cloud Styling.html

```diff
@@ -124,6 +124,26 @@
     pointSize : "5"
 });
 
+addStyle('Sign', {
+    color : "color() * sign(${temperature})",
+    pointSize : "5"
+});
```

Comment:
Just a reminder about this comment. `fract` sort of applies here too. Even just `sign(${temperature} = 0.5)` may be sufficient.